projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -2606,6 +2610,66 @@ private boolean showVersion(List<String> options, List<String> parameters) {
     return true;
   }
 
+  private boolean syncTestrigsSyncNow(List<String> options, List<String> parameters) {
+    if (!isValidArgument(options, parameters, 1, 1, 1,
+            Command.SYNC_TESTRIGS_SYNC_NOW)) {
+      return false;
+    }
+    if (!isSetContainer(true)) {
+      return false;
+    }
+
```

Comment:
do you have the `google-java-format` plugin installed that Ari set up to keep everything consistent? Looks like this file is fairly malformed w.r.t. that format.

Reply:
Fixed (by you:)